examples/magento-graphcms/pages/checkout/added.tsx

```diff
@@ -43,11 +43,8 @@ function CheckoutAdded() {
       filterNonNullableKeys(
         crosssels.data?.products?.items?.[0]?.crosssell_products ??
           crosssels.previousData?.products?.items?.[0]?.crosssell_products,
-      ).filter(
-        (item) =>
-          items.every((i) => i.product.sku !== item.sku) && item.stock_status === 'IN_STOCK',
-      ),
-    [crosssels.data?.products?.items, crosssels.previousData?.products?.items, items],
+      ).filter((item) => item.stock_status === 'IN_STOCK'),
```

Comment:
We're removing the check of the SKU here, it seems to be fine, but can't guess what this actually changes? What changed here?